tests/test_kimi_bridge_server.py

```
@@ -0,0 +1,184 @@
+
+    The key is patching ``KimiBrowserDriver`` *inside* app.py's namespace —
+    that's the name the lifespan closure uses, not the one in browser_driver.py.
+    """
+    monkeypatch.setenv("KIMI_BRIDGE_TOKEN", "test-secret")
+
+    from fastapi.testclient import TestClient
+    import services.kimi_bridge_server.app as app_mod
+
+    # Patch the name as it appears in the app module (``from .browser_driver import …``)
+    monkeypatch.setattr(app_mod, "KimiBrowserDriver", lambda: fake_driver)
+    app_mod._BRIDGE_TOKEN = "test-secret"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "_BRIDGE_TOKEN"

(S105)